src/components/UI/atoms/RatingsCount/RatingsCount.spec.js

```diff
@@ -10,7 +10,9 @@ afterEach(cleanup);
 const setup = (color) => {
   const utils = render(
     <RatingsCount
-      color={color ? 'darkPurple' : 'pink'}>
+      color={color ? 'darkPurple' : 'pink'}
```

Comment:
I don't understand why there is conditionally logic to render the color shouldn't we just assign a color